core/resources/shaders/point.fs

```diff
@@ -48,8 +48,16 @@ void main(void) {
             float c2 = circle(uv, vec2(0.0), circleRadiusOut);
             color = vec4(vec3(c1) * v_color.rgb, c2 * v_alpha * v_color.a);
         #else
+            // premultiplied texture color
             vec4 texColor = texture2D(u_tex, v_texcoords);
-            color = vec4(texColor.rgb * v_color.rgb, v_alpha * texColor.a * v_color.a);
+            //color = vec4(texColor.rgb * v_color.rgb, texColor.a * v_color.a);
+
+            // Correct? v_color as multipler - to say how much of each texture channel is used?
+            color = texColor * v_color;
+            color *= v_alpha;
```

Comment:
Any of the styles can sample from textures, sprites aren't the only place we would need to un-pre-multiply. 

There is a bigger problem to changing color representation though: User-defined shader code can interact with sampled color values. In the current pipeline, none of the color values available for a user to interact with are pre-multiplied, and changing that could certainly cause visual differences. We could do the necessary arithmetic on pre-defined color variables before injection points, but that won't help if a user calls a sampler in their own code. 

I do think that alpha-pre-multiplied textures are the "right" way to handle transparency, but I also want to make sure it won't disrupt existing shaders in styles or create a divergence with tangram-js.